components/database/mysql-common/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Jim Klimov
+# Copyright 2021 Andreas Wacknitz
+#
+include ../../../make-rules/shared-macros.mk
+
+# Beside the p5m file, there is really nothing to this component which just
+# delivers a user and group actions as a common ground for database servers of mysql heritage.
+COMPONENT_NAME=				mysql-common
+COMPONENT_VERSION=			1.0
```

Comment:
I think we should bump this just to be safe.

Reply:
@AndWac This we still need to so. With incorporations we nail the version so buildnum helps, but unless we bump `COMPONENT_VERSION` Old versions could still be installed.